libcxx/include/__vector/vector.h

```diff
@@ -116,6 +426,12 @@ class vector {
   using reverse_iterator       = std::reverse_iterator<iterator>;
   using const_reverse_iterator = std::reverse_iterator<const_iterator>;
 
+  using __base::__invariants;
+  using __base::back;
+  using __base::capacity;
+  using __base::empty;
+  using __base::size;
```

Comment:
Although users aren't allowed to take the address of member functions from standard library types, there seems to be an endless supply of build breaks when I do this. The problem is that `&std::vector<T>::size` no longer exists, since the function is a member of `__vector_layout`.

Due to the overwhelming number of build breaks, I'm going to defer doing this to a later date, when a library-wide policy can be enacted. I'll update the PR to have `std::vector::size` call `__vector_layout::size`, etc. in a little bit, so that this PR isn't blamed for breaking ~everyone.